unixhttpc/unixhttpc.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io"
+	"net"
+	"net/http"
+	"os"
+	"strings"
+)
+
+func main() {
+	post := flag.String("d", "", "data to POST")
+	help := flag.Bool("h", false, "usage help")
+	flag.Parse()
+
+	if *help || len(flag.Args()) != 2 {
+		fmt.Fprintln(os.Stderr, "usage:", os.Args[0], "[-d data] /path.socket /uri")
```

Comment:
Lets put a description of what this command does and where people can find more information about the wifi-ap snap -> https://docs.ubuntu.com/core/en/stacks/network/wifi-ap/docs/index